### PR TITLE
Update Linux docs

### PR DESCRIPTION
**Pull request by paulcsmith**

### CODE REVIEW

File: source/guides/installing.html.md.erb [hunk 1]
```diff
@@ -27,7 +27,15 @@ Make sure you have home-brew, then run `brew install lucky`
...
-* Do a bunch of other crap
+Please help contribute to this section by [editing this page on GitHub](https://github.com/luckyframework/website/blob/master/source/guides/installing.html.md.erb).
+
+These steps may vary depending on your Linux distribution
+
+* `git clone` the CLI repo at https://github.com/luckyframework/cli
+* cd into the newly cloned directory
+* Run `crystal deps`
+* Run `crystal build src/lucky.cr --release --no-debug`
+* Move the generated `lucky` binary to `usr/bin`
```
Comment (BlakeWilliams) [inline source/guides/installing.html.md.erb]: `/usr/bin`?

Comment (paulcsmith) [reply source/guides/installing.html.md.erb]: Good catch!

File: source/guides/installing.html.md.erb [hunk 2]
```diff
@@ -27,7 +27,15 @@ Make sure you have home-brew, then run `brew install lucky`
...
-* Do a bunch of other crap
+Please help contribute to this section by [editing this page on GitHub](https://github.com/luckyframework/website/blob/master/source/guides/installing.html.md.erb).
+
+These steps may vary depending on your Linux distribution
+
+* `git clone` the CLI repo at https://github.com/luckyframework/cli
+* cd into the newly cloned directory
+* Run `crystal deps`
+* Run `crystal build src/lucky.cr --release --no-debug`
+* Move the generated `lucky` binary to `/usr/bin`
```
Comment (mike-burns) [inline source/guides/installing.html.md.erb]: Probably not `/usr/bin`. Do you mean "move it into your `$PATH`"? Alternatively, `/usr/local/bin` is probably on everyone's path.

File: source/guides/installing.html.md.erb [hunk 3]
```diff
@@ -27,7 +27,15 @@ Make sure you have home-brew, then run `brew install lucky`
 
 ## On Linux machines:
 
-* Do a bunch of other crap
+Please help contribute to this section by [editing this page on GitHub](https://github.com/luckyframework/website/blob/master/source/guides/installing.html.md.erb).
+
+These steps may vary depending on your Linux distribution
```
Comment (mike-burns) [inline source/guides/installing.html.md.erb]: They will?

**Review by mike-burns — COMMENTED** [review]
```
~/cli% /usr/bin/lucky 

Missing tasks.cr file

Try this...

  ▸ Change to the directory with the task.cr file,
    usually your project root
  ▸ If this is a new project, run lucky init to
    create a default tasks.cr file
~/cli% ls **/task.cr
src/lucky_cli/task.cr
```

File: source/guides/installing.html.md.erb (hunk 3 above)
Comment (paulcsmith) [reply source/guides/installing.html.md.erb]: I'm not sure. That's why I said "may" lol. I know that some people have had to install extra packages to be able to build their crystal packages: https://amberframework.org/guides/getting-started/Installation/README.md#installation

Maybe not worth adding that caveat until I know there are issues?

File: source/guides/installing.html.md.erb (hunk 2 above)
Comment (paulcsmith) [reply source/guides/installing.html.md.erb]: Ah that's what I meant. `/usr/local/bin` is better.

I'll change it to "Move the generated `lucky` binary to your path. Most of the time you can move it to `/usr/local/bin` and it should work."

WDYT?

**Comment by paulcsmith** [comment]
@mike-burns Perfect! Thanks for the feedback and for trying it out

File: source/guides/installing.html.md.erb (hunk 3 above)
Comment (mike-burns) [reply source/guides/installing.html.md.erb]: Yeah I'd wait until there are issues. The devs would need `build-essential`, which all devs need anyway.

File: source/guides/installing.html.md.erb (hunk 2 above)
Comment (mike-burns) [reply source/guides/installing.html.md.erb]: Perfect!

File: source/guides/installing.html.md.erb (hunk 3 above)
Comment (paulcsmith) [reply source/guides/installing.html.md.erb]: 👍

**Pull request merged**